app/assets/javascripts/oxalis/view/right-menu/mapping_info_view.js

```diff
@@ -169,24 +196,85 @@ class MappingInfoView extends React.Component<Props> {
     );
   }
 
+  handleChangeMapping = (mappingName: string): void => {
+    const progressCallback = createProgressCallback({ pauseDelay: 500, successMessageDelay: 2000 });
+    Model.getSegmentationLayer().setActiveMapping(mappingName, progressCallback);
+    this.handleSetMappingEnabled(true);
+  };
+
+  async refreshLayerMappings() {
+    if (this.state.didRefreshMappingList) {
+      return;
+    }
+    this.setState({ isRefreshingMappingList: true });
+    const layerName = "segmentation";
```

Comment:
Shouldn't this be `this.props.segmentationLayer.name`?

Reply:
My hope was that this will allow mappings even when there is a new volume tracing layer, however, this doesn't work, since the actual mappings are downloaded via an url which is defined somewhere also (and again with the segmentation layer's name). I could change it there too, but in general, this is brittle, since a segmentation layer can have different names on disk, as well. So, I went back to using the actual layer name here. It's still an improvement to before.

We should rethink how we want to handle the disk segmentation layer VS volume tracing layer in the future.